### DISCUSSION TITLE
Welcome to SuperSlicer Discussions!

### DISCUSSION THREAD

**Discussion by supermerill** [supermerill/SuperSlicer#781] [General]
## 👋 Welcome!
  Discussions is a place to connect with other members of our community.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by RunboyUSA** [comment]
Wanted to let you know that I created a SuperSlicer support group on Facebook.
I needed some help and check for some support on Facebook and didn't find any; so I created it.
If you are interested, you more than are welcome to join.
This is an amazing Slicer.
Thanks for all the work you are doing.

**Reply by supermerill** [reply]
Thank you, but I'm not using facebook at all.

**Comment by sillywan** [comment]
I just unzipped and tried to run version 2.3.57.11_64bit_220213 and it will not boot. Has anyone else had this problem? Is there anyone out there?

**Reply by supermerill** [reply]
It's not currently signed (i'm working on it), so maybe your antivirus don't let you run it?
On a virgin windows, you have to select "more information" and " execute" the first time.

**Comment by sillywan** [comment]
If I leave the icons in their home directory they work.
But if I move the Icons to the desk top they try to start up and fail.
So, basicly the Shortcuts to start the program only work in the unzipped
home directory. But it works! It's just a bit of an effort to start it up.
